### PR TITLE
Bump deps

### PR DESCRIPTION
**Pull request by AlekSi**
## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [x] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: tools/go.mod [hunk 1]
```diff
@@ -4,14 +4,14 @@ go 1.19
 
 require (
 	github.com/BurntSushi/go-sumtype v0.0.0-20221020234012-480526a59796
-	github.com/go-task/task/v3 v3.17.0
-	github.com/golangci/golangci-lint v1.50.1 // sync with GitHub Actions workflow
```
Comment (chilagrow) [inline tools/go.mod:8]: do we want to keep this comment?

**Review by chilagrow — COMMENTED** [review]
There is linter warning :thinking: interesting as the linter version wasn't updated

File: tools/go.mod (hunk 1 above)
Comment (AlekSi) [reply tools/go.mod:8]: No, it was about https://github.com/golangci/golangci-lint-action, and it is very unlikely we would use it – it does not provide enough configurability

**Comment by AlekSi** [comment]
That's because `golang.org/x/tools` was updated, that includes `vet` updates. PTAL

**Review by w84thesun — APPROVED** [review]

**Review by noisersup — APPROVED** [review]

**Pull request merged**